src/data/move.ts

```diff
@@ -10074,8 +10166,10 @@ export function initMoves() {
       .attr(IvyCudgelTypeAttr)
       .attr(HighCritAttr)
       .makesContact(false),
-    new AttackMove(Moves.ELECTRO_SHOT, Type.ELECTRIC, MoveCategory.SPECIAL, 130, 100, 10, 100, 0, 9)
-      .attr(ElectroShotChargeAttr)
+    new ChargingAttackMove(Moves.ELECTRO_SHOT, Type.ELECTRIC, MoveCategory.SPECIAL, 130, 100, 10, -1, 0, 9)
+      .chargeText(i18next.t("moveTriggers:absorbedElectricity", { pokemonName: "{USER}" }))
+      .chargeAttr(StatStageChangeAttr, [ Stat.SPATK ], 1, true)
+      .chargeAttr(WeatherInstantChargeAttr, [ WeatherType.RAIN, WeatherType.HEAVY_RAIN ])
       .ignoresVirtual(),
```

Comment:
```suggestion
      .ignoresVirtual()
      .edgecase(), // Should be boosted by Sheer Force without removing the +1 Sp Atk boost
```
https://bulbapedia.bulbagarden.net/wiki/Electro_Shot_(move)

> Electro Shot's damage is boosted by [Sheer Force](https://bulbapedia.bulbagarden.net/wiki/Sheer_Force_(Ability)) despite not suppressing any additional effects.

Reply:
oh, for some reason it wasn't listed on Sheer Force's page, so I changed its chance to `-1`. I can just change it back to what it used to be.

Is Meteor Beam treated the same way? I don't see any mention of Sheer Force's interaction with it on [Bulba](https://bulbapedia.bulbagarden.net/wiki/Meteor_Beam_(move))